lib/PipelineC/PipelineC.cpp

```diff
@@ -69,6 +69,10 @@ bool rp_initialize(int argc,
   return true;
 }
 
+bool rp_shutdown() {
+  return true;
```

Comment:
Wat

Reply:
In one commit I introduce it and it does nothing. In a subsequent commit I implement stuff that needs to be performed on shutdown.

This looks weird but I was trying to avoid introducing two conceptually different things in one commit.

Reply:
ACK, in general don't push this approach too far. We do this kind of things only if reviewing the whole logic altogether is very difficult. Let's not go down slippery slope of infinitely splitting commits.

Consider that in general I don't accept fixup commits (commits fixing previously introduced things).